src/QirRuntime/test/Standalone/InputTypes/compiler/QirCore.qs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If you generate the QIR from Q# project, you shouldn't need this file.

Reply:
Yes, I started doing some of this work before it was possible to do that but now that it is available I'll merge main and delete these files that are no longer needed :)

Reply:
Using a Q# project and the `QirGeneration` property now.